src/partials/our_products.html

```diff
@@ -0,0 +1,7 @@
+<section>
```

Comment:
gdybyś jeszcze nazwała tę sekcję byłoby super. Co do tych zdjęć, na razie zrób jak na makiecie, byśmy mieli podstawę, by  ukazać projekt. Z tego co widzę, powinno tam być js, a jako iż my go nie mieliśmy to się tym nie martw. Jak widzisz na dole Twojej sekcji jest 8. kropek, spróbuj je odwzorować za pomocą ul, a w pierwszym daj klase active-our-products i wystylizuj go na pomaranczowo. Miłego dnia (:

Reply:
Dorzuciłam nazwę klasy chwilę po tym jak wysłałam pull requesta, więc klasa powinna być już widoczna. 
Co do zdjęć - zrobię na sam początek te 4, które widać, ostyluję i tyle. Sliderem pobawię się jak już wstępnie ta sekcja będzie jakoś wyglądała (myślę, że dzisiaj ją skończę) ;)